packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2196,7 +2196,7 @@ export class StripeHelper {
 
     const abbrevProductNew = await this.expandAbbrevProductForPlan(planNew);
     const {
-      interval: productPaymentCycle,
+      interval: productPaymentCycleNew,
```

Comment:
I'm not sure if I needed to change this.

Reply:
@biancadanforth @chenba, I am not sure if I need to update `productPaymentCycle` in `stripe.ts`, can you please review and let me know if this should be included in the fix? Thank you! 🙂

Reply:
The renaming make sense.  But I cannot find where `productPaymentCycleOld` for the template is coming from.

Reply:
Thanks for your feedback, @chenba! I have updated the PR to include `productPaymentCycleOld` for the template. When you have a moment, can you please take another look? Thanks again!